spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigDataLocationResolver.java

```diff
@@ -84,19 +83,12 @@ protected void registerBeans(ConfigDataLocationResolverContext resolverContext,
 
 	}
 
-	private KubernetesClient registerConfigAndClient(ConfigurableBootstrapContext bootstrapContext,
+	private KubernetesClient registerClient(ConfigurableBootstrapContext bootstrapContext,
 			KubernetesClientProperties kubernetesClientProperties) {
 		Config config = new Fabric8AutoConfiguration().kubernetesClientConfig(kubernetesClientProperties);
-		registerSingle(bootstrapContext, Config.class, config, "fabric8Config");
```

Comment:
we do not need to register the `Config`, as it is only needed in order to create the `KubernetesClient`, that one though, needs to be registered.